test/integration/tests/backstop.ts

```diff
@@ -0,0 +1,93 @@
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { ethers } from 'hardhat';
+import { NamedAddresses, NamedContracts } from '@custom-types/types';
+import { expectApprox, resetFork, time } from '@test/helpers';
+import proposals from '@test/integration/proposals_config';
+import { TestEndtoEndCoordinator } from '@test/integration/setup';
+
+const e18 = ethers.constants.WeiPerEther;
+const uintMax = ethers.constants.MaxUint256;
+const toBN = ethers.BigNumber.from;
+
+before(async () => {
+  chai.use(CBN(ethers.BigNumber));
+  chai.use(solidity);
+  await resetFork();
+});
+
+describe.only('e2e', function () {
+  let contracts: NamedContracts;
+  let contractAddresses: NamedAddresses;
+  let deployAddress: string;
+  let e2eCoord: TestEndtoEndCoordinator;
+  let doLogging: boolean;
+
+  const tenPow18 = toBN('1000000000000000000');
+
+  before(async function () {
+    // Setup test environment and get contracts
+    const version = 1;
+    deployAddress = (await ethers.getSigners())[0].address;
+    if (!deployAddress) throw new Error(`No deploy address!`);
+
+    doLogging = Boolean(process.env.LOGGING);
+
+    const config = {
+      logging: doLogging,
+      deployAddress: deployAddress,
+      version: version
+    };
+
+    e2eCoord = new TestEndtoEndCoordinator(config, proposals);
+
+    doLogging && console.log(`Loading environment...`);
+    ({ contracts, contractAddresses } = await e2eCoord.loadEnvironment());
+    doLogging && console.log(`Environment loaded.`);
+  });
+
+  describe('TribeMinter', async function () {
+    it('mint TRIBE', async function () {
+      const { tribeMinter, tribe } = contracts;
+      const tribeSupply = await tribe.totalSupply();
+      const balanceBefore = await tribe.balanceOf(deployAddress);
+
+      await tribeMinter.mint(deployAddress, '100000');
```

Comment:
Maybe I'm missing something here - but why is the deploy address allowed to mint Tribe for free?

Reply:
Deploy address is a governor in our integration test setup